documentation/modules/mirrormaker2/proc-mirrormaker-replication.adoc

```diff
@@ -116,6 +116,62 @@ spec:
         checkpoints.topic.replication.factor: 1 <25>
     topicsPattern: ".*" <26>
     groupsPattern: "group1|group2|group3" <27>
+  resources: <28>
+    requests:
+      cpu: "1"
+      memory: 2Gi
+    limits:
+      cpu: "2"
+      memory: 2Gi
+  logging: <29>
+    type: inline
+    loggers:
+      connect.root.logger.level: "INFO"
+  readinessProbe: <30>
+    initialDelaySeconds: 15
+    timeoutSeconds: 5
+  livenessProbe:
+    initialDelaySeconds: 15
+    timeoutSeconds: 5
+  jvmOptions: <31>
+    "-Xmx": "1g"
+    "-Xms": "1g"
+  image: my-org/my-image:latest <32>
+  template: <33>
+    pod:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                  - key: application
+                    operator: In
+                    values:
+                      - postgresql
+                      - mongodb
+              topologyKey: "kubernetes.io/hostname"
+    connectContainer: 1
```

Comment:
this `1` doesn't make sense here. It should be a callout I guess

Reply:
Must have got lost in the merge